### PR TITLE
[YUNIKORN-2474] Delete unused variable and methods

### PR DESCRIPTION
**Pull request by amikai**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - http://yunikorn.apache.org/community/how_to_contribute   

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [X] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2474

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: test/e2e/admission_controller/admission_controller_suite_test.go [hunk 1]
```diff
@@ -199,7 +199,7 @@ var _ = BeforeSuite(func() {
 
 	annotation = "ann-" + common.RandSeq(10)
 	yunikorn.EnsureYuniKornConfigsPresent()
-	yunikorn.UpdateConfigMapWrapper(oldConfigMap, "", annotation)
+	yunikorn.UpdateConfigMapWrapper(oldConfigMap, "")
```
Comment (brandboat) [inline test/e2e/admission_controller/admission_controller_suite_test.go:202]: now annotation in Line 58 become unused, could you help remove it too ? Same thing in other places.

Comment (amikai) [reply test/e2e/admission_controller/admission_controller_suite_test.go:200]: done in b6c835adb62b7478e829cbe2e14f617209ab051e

**Review by brandboat — APPROVED** [review]
LGTM, many thanks !

File: test/e2e/admission_controller/admission_controller_suite_test.go [hunk 2]
```diff
@@ -197,9 +196,8 @@ var _ = BeforeSuite(func() {
 	kubeClient = k8s.KubeCtl{}
 	Expect(kubeClient.SetClient()).To(BeNil())
 
-	annotation = "ann-" + common.RandSeq(10)
 	yunikorn.EnsureYuniKornConfigsPresent()
-	yunikorn.UpdateConfigMapWrapper(oldConfigMap, "", annotation)
```
Comment (chia7712) [inline test/e2e/admission_controller/admission_controller_suite_test.go:202]: Could you please trace the git to find out which commit makes it unused? I guess https://github.com/apache/yunikorn-k8shim/commit/ec9d10d60dc4e8529a3ffa346502d2242bc7d4d0

File: test/e2e/wrappers.go [hunk 3]
```diff
@@ -36,61 +32,6 @@ import (
 
 var k = k8s.KubeCtl{}
 
-func UpdateConfigMapWrapper(oldConfigMap *v1.ConfigMap, schedPolicy string, annotation string) {
```
Comment (chia7712) [inline test/e2e/wrappers.go:39]: those two methods were introduced by https://github.com/apache/yunikorn-k8shim/commit/55e8148863a00b28be243f6e9ad38787726e065f. And they are similar to `yunikorn. UpdateConfigMapWrapper` and `yunikorn.RestoreConfigMapWrapper`. Was the error caused by fat finger or it intended to do something difference?

Comment (amikai) [reply test/e2e/wrappers.go:39]: Hi, @manirajv06, I noticed that this parameter is not being used, so I am starting the deletion process. 
However, I speculate that this design might involve assigning an annotation as a unique identifier during the UpdateConfigMapWrapper operation, and then using this annotation to retrieve it during the use of RestoreConfigMapWrapper.

This design could be aimed at preventing multiple tests from simultaneously using UpdateConfigMapWrapper, but I'm not entirely certain.

I guess you accidentally forgot it and then leave the annotation parameter here. Could you help me confirm that?

Comment (brandboat) [reply test/e2e/wrappers.go:39]: > those two methods were introduced by https://github.com/apache/yunikorn-k8shim/commit/55e8148863a00b28be243f6e9ad38787726e065f. And they are similar to yunikorn. UpdateConfigMapWrapper and yunikorn.RestoreConfigMapWrapper. Was the error caused by fat finger or it intended to do something difference?

gentle ping @wilfred-s, I find myself uncertain here and would highly value your insights. Could you please share your thoughts?

Comment (chia7712) [reply test/e2e/wrappers.go:39]: I'd like to ship this PR, so @amikai could you please file a follow-up PR to trace those similar functions? also this PR does NOT remove those similar functions

WDYT?

Comment (amikai) [reply test/e2e/wrappers.go:39]: Sounds good to me. I will create another PR to remove the unused variable, and keep these "similar functions" to this PR for discussion.

Comment (amikai) [reply test/e2e/wrappers.go:39]: Sounds good to me. I will create another PR to remove the unused variable, and keep these “similar functions” to this PR for discussion

**Review by pbacsko — APPROVED** [review]
+1 LGTM

please rebase the master to see a green build again

**Comment by chia7712** [comment]
@amikai could you please rebase code?

**Comment by amikai** [comment]
Hi @chia7712 already rebased to master.

**Review by chia7712 — APPROVED** [review]
will merge it after QA pass

**Pull request closed without merging**